deploy/ray-operator/controllers/raycluster_controller.go

```diff
@@ -0,0 +1,238 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	mapset "github.com/deckarep/golang-set"
+	"github.com/go-logr/logr"
+	_ "k8s.io/api/apps/v1beta1"
+	rayiov1alpha1 "ray-operator/api/v1alpha1"
+	"ray-operator/controllers/common"
```

Comment:
The package name here is not good practice. Can we use something like `github.com/ray-project/ray`?

Reply:
Sure, I am working arranging these.